### DISCUSSION TITLE
Verifying idToken in Backend when using Auth0

### DISCUSSION THREAD

**Discussion by RoniFinTech** [Web3Auth/Web3Auth#914] [Help]
Hi,

Should I verify the idToken in my backend with openlogin or directly go to auth0?

Thanks!

**Comment by shahbaz17** [comment]
Hey @RoniFinTech 

Is `isToken` here returned from web3auth? Then, yes, you can verify the idToken. Please take a look at https://web3auth.io/docs/server-side-verification/social-login-users#verifying-jwt-token-idtoken
